interface/i18n/mist.en.i18n.json

```diff
@@ -71,7 +71,8 @@
                 "toFront": "Bring All to Front"
             },
             "help": {
-                "label": "Help"
+                "label": "Help",
+                "reportBug": "Report a bug on Github"
```

Comment:
i created that reportbug thing. Maybe we should rephrase to "Report an issue on Github"?